### DISCUSSION TITLE
Repo added to list but star disappear

### DISCUSSION THREAD

**Discussion by AlrikRr** [community/community#20205] [Other Feature Feedback, Questions, & Ideas]
Hello There !

I saw a graphical issue when adding a repo to a list, it happens sometimes when I let the repo page open for a while or restore the session of my browser.

The star icon is blank like if the repo was not starred but if I expend the list I can see the repo inside of a created list.
For some repo I was like "Oh but I swear I added it into my list !" and it was.

Here is a screenshot of a starred repo without the start icon:
![Capture](https://user-images.githubusercontent.com/32389491/177986799-6c066d19-2c75-4a37-8c79-90e2693fdd03.PNG)

Has anyone else experienced this problem ? 

Sorry if this is a duplicate !

**Comment by Josephglz** [comment]
Hello! The same thing happened to me, what I did was just remove it from the list, click on the star and then add it to a specific list, It really is only an aesthetic detail since the repository is added in the same way to the lists or "favorites"
